File: IsraelHiking.Web/android/app/src/main/java/com/mapeak/car/CarStoreKeys.kt
package com.mapeak.car

/**
 * Well-known keys exchanged through [CarStore]. STYLE/ROUTE/CONFIG mirror the string keys sent from
 * the web layer via the Car Capacitor plugin and must not change. LOCATION/CONNECTED are produced
 * on the native side and broadcast to listeners. ZOOM/LAST_LAT/LAST_LNG are persisted-only and are
 * never part of the listener contract.
 */
object CarStoreKeys {
    const val STYLE = "style"
    const val ROUTE = "route"
    const val CONFIG = "config"
    const val LOCATION = "location"
    const val ZOOM = "zoom"
    const val LAST_LAT = "last_lat"
    const val LAST_LNG = "last_lng"
}

File: IsraelHiking.Web/ios/App/App/Car/CarStoreKeys.swift
import Foundation

/**
 * Well-known keys exchanged through `CapacitorStore`. style/route/config mirror the string keys
 * sent from the web layer via the ReactivePreferences plugin and must not change. location is
 * produced on the native side and broadcast to listeners. zoom/lastLat/lastLng are persisted-only
 * and are never part of the listener contract. Mirrors `CarStoreKeys.kt` on Android.
 */
enum CarStoreKeys {
    static let style = "style"
    static let route = "route"
    static let config = "config"
    static let location = "location"
    static let zoom = "zoom"
    static let lastLat = "last_lat"
    static let lastLng = "last_lng"
}
